### PR TITLE
Update eBay

### PR DESCRIPTION
**Pull request by gingerbeardman**
adds new software 2FA using app feature "eBay Verify"

See #3858 for proof

### CODE REVIEW

**Comment by RichJeanes** [comment]
Please add an exception to the entry, similar to Steam, stating that it requires the use of a proprietary app.

**Comment by gingerbeardman** [comment]
Done!

**Comment by kmpoppe** [comment]
Could you check, whether there is a new documentation link in the help
pages that could replace my PR #3811?

// Kai

Matt Sephton <notifications@github.com> schrieb am Fr., 17. Mai 2019, 17:47:

File: _data/retail.yml [hunk 1]
```diff
@@ -131,8 +131,11 @@ websites:
       img: ebay.png
       tfa: Yes
       sms: Yes
+      software: Yes
+      exceptions:
+        text: "Software implementation requires use of the eBay app."
 
-    - name: ebay Kleinanzeigen
```
Comment (RichJeanes) [inline _data/retail.yml]: These files are whitespace-sensitive. Please add this back in to fix your changes.

**Comment by gingerbeardman** [comment]
nothing visible, so I've asked

(I actually prefer SMS verification as I can receive it on my Mac when my iPhone is out of reach)

**Comment by kmpoppe** [comment]
@gingerbeardman I think you broke the .yml because the eBay Kleinanzeigen start is now not in line anymore. Could you please correct that?

**Comment by gingerbeardman** [comment]
@kmoppe fixed, apologies. Doc added (sent to me by one of the team on twitter [here](https://twitter.com/itickr/status/1129442789064790017?s=21)) 

though it's still not ideal so I've politely asked for better 2FA docs.

**Review by kmpoppe — APPROVED** [review]
Looks good!

**Review by RichJeanes — APPROVED** [review]

**Pull request merged**